.eslintignore

```diff
@@ -0,0 +1,2 @@
+/bower_components-1.x/*
```

Comment:
The [status-indicator](https://github.com/BrightspaceUI/status-indicator) is a good example for eslint configuration, rules, and tasks.